src/mission_companion.cpp

```diff
@@ -185,28 +185,28 @@ bool talk_function::outpost_missions(npc *p, std::string id, std::string title)
     std::string entry, entry_aux;
 
     if (id == "SCAVENGER"){
-        col_missions["Assign Scavenging Patrol"] = "Profit: $25-$500\nDanger: Low\nTime: 10 hour missions\n \n"
+        col_missions["Assign Scavenging Patrol"] = _("Profit: $25-$500\nDanger: Low\nTime: 10 hour missions\n \n"
```

Comment:
I see, and I'm not sure, that should work, but it's possible that the
gettext string extraction tool will not handle concatenated strings
properly.